src/components/elements/annotationSidePanel/components/annotationWizard/AnnotationWizard.tsx

```diff
@@ -325,11 +319,11 @@ const AnnotationWizard = (props: Props) => {
                                                 }}
                                             >
                                                 <p>
-                                                    Exit
+                                                    {isIdentificationOrGeoreference ? `< Back to annotations` : 'Exit'}
                                                 </p>
                                             </Button>
                                         </Col>
-                                        {!!selectedIndex &&
+                                        {(!!selectedIndex && !isIdentificationOrGeoreference) &&
```

Comment:
maybe make this name more generic, like isClassAnnotationFlow, for when you add other kinds of custom flows?